### PR TITLE
feat: Change inbound prompting

### PR DESCRIPTION
**Pull request by GDamyanov**
Feat for #2072 

- Add prompting for Change Inbound
- Add CLI command for Change Inbound

### CODE REVIEW

**Review by testojs — APPROVED** [review]
* code looks gode
* test coverage is good
* change set is correct
* not tested manually - to be extensively tested by @nnikovSAP

File: packages/adp-tooling/src/translations/adp-tooling.i18n.json [hunk 1]
```diff
@@ -8,21 +8,35 @@
         "maxAgeTooltip": "(Optional) Enter the time in seconds it takes for a cached response of the service to expire.",
         "oDataAnnotationSourceURILabel": "OData Annotation Data Source URI",
         "oDataAnnotationSourceURITooltip": "Server side annotations have been detected for the selected OData service. If needed, enter new annotation data source URI  for the changed service",
+        "oDataURIInfoLabel": "Have in mind that common OData Service path prefixes are '{{- v2Prefix}}' or '{{- v4Prefix}}'",
+        "oDataURIV2InfoLabel": "Have in mind that you are trying to replace OData {{versionOne}} Service with OData {{versionTwo}} Service",
```
Comment (tobiasqueck) [inline packages/adp-tooling/src/translations/adp-tooling.i18n.json:12]: I don't see oDataURIInfoLabel, oDataURIV2InfoLabel or inboundId used anywhere. Why do we have them here? Or did I miss a reference?

Comment (GDamyanov) [reply packages/adp-tooling/src/translations/adp-tooling.i18n.json]: Fixed in [23549bc](https://github.com/SAP/open-ux-tools/pull/2167/commits/23549bc071b644cff18414ef2f2b3f32fcdeb5bb)

**Review by tobiasqueck — APPROVED** [review]
* changeset ok
* code adheres to coding standards
* did not test locally

**Review by testojs — APPROVED** [review]

**Pull request merged**